package.json

```diff
@@ -44,25 +46,31 @@
     "Snippets"
   ],
   "devDependencies": {
-    "@types/mocha": "^5.2.5",
-    "@types/node": "^6.0.40",
-    "gulp": "^4.0.0",
+    "@types/fs-extra": "^7.0.0",
+    "@types/md5": "^2.1.33",
+    "@types/mocha": "^5.2.6",
+    "@types/node": "^10.14.16",
+    "@types/vscode": "^1.37.0",
+    "@types/which": "^1.3.1",
+    "@types/xml2js": "^0.4.4",
```

Comment:
We don't need the following dependencies:   
"@types/fs-extra": "^7.0.0",
"@types/md5": "^2.1.33",
"@types/which": "^1.3.1",
"@types/xml2js": "^0.4.4",

Reply:
Good catch. It was a copy/paste mistake. It should be fixed now. I have just one question, how have you seen the trouble (have you some error lint you could have to see unused dependencies?)